server/tests/src/test/java/org/infinispan/server/test/core/ldap/RemoteLdapServer.java

```diff
@@ -31,10 +31,10 @@ public void start(String keystoreFile, String[] initLDIFs) throws Exception {
       URI ldapUri = new URI(System.getProperty(TEST_LDAP_URL));
       LdapNetworkConnection connection = new LdapNetworkConnection(ldapUri.getHost(), ldapUri.getPort());
       connection.setTimeOut(60);
-      connection.bind(System.getProperty(TEST_LDAP_HOST_USER), System.getProperty(TEST_LDAP_HOST_PASSWORD));
+      connection.bind(System.getProperty(TEST_LDAP_PRINCIPAL), System.getProperty(TEST_LDAP_CREDENTIAL));
 
       for (String initLDIF : initLDIFs) {
-         if (!initLDIF.contains("-dn.ldif")) {
+         if (!initLDIF.contains("-ou.ldif")) {
```

Comment:
Could you add a comment to explain why `-ou` files are handled differently?

Reply:
Explained in the code